portal/config/eproms/Organization.json

```diff
@@ -1106,6 +1106,13 @@
         {
           "timezone": "America/Chicago",
           "url": "http://hl7.org/fhir/StructureDefinition/user-timezone"
+        },
+        {
+          "research_protocols": [
+			{"name": "IRONMAN v2", "retired_as_of": "2018-07-20T05:00:00Z"},
```

Comment:
Why is the `retired_as_of` date so far in the past?  This will potentially lead to broken logic, as we assume only a single QB may straddle the transition from old to new.